pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[project]
+authors = [
+    {name = "Webkom", email = "webkom@abakus.no"},
+]
+dependencies = [
+    "django==4.0.8",
+    "ipython==8.7.0",
+    "psycopg2-binary==2.9.5",
+    "unicode-slugify==0.1.5",
+    "pyyaml==6.0",
+    "redis==4.4.0",
+    "hiredis==2.1.0",
+    "certifi==2022.12.7",
+    "elasticsearch==7.16.2",
+    "celery==5.2.7",
+    "stripe==3.4.0",
+    "boto3==1.26.31",
+    "beautifulsoup4==4.11.1",
+    "bleach==5.0.1",
+    "requests==2.28.1",
+    "ldap3==2.9.1",
+    "google-api-python-client==2.70.0",
+    "oauth2client>=4.1.3",
+    "premailer==3.10.0",
+    "libthumbor==2.0.2",
+    "django-health-check==3.17.0",
+    "djangorestframework==3.14.0",
+    "drf-jwt==1.19.2",
+    "djangorestframework-camel-case==1.3.0",
+    "django-extensions==3.2.1",
+    "django-autoslug==1.9.8",
+    "django-oauth-toolkit==1.6.3",
+    "django-cors-headers==3.13.0",
+    "django-mptt==0.14.0",
+    "django-filter==22.1",
+    "django-environ==0.9.0",
+    "django-redis==5.2.0",
+    "django-ipware==4.0.2",
+    "django-phonenumber-field[phonenumberslite]==7.0.1",
+    "django-push-notifications==3.0.0",
+    "django-ical==1.8.3",
+    "icalendar==5.0.3",
+    # Channels
+    "channels==3.0.5",
+    "channels-redis==4.0.0",
+    "daphne==3.0.2",
+    "flatbuffers==22.12.6",
+    # Logging/errors
+    "structlog==21.5.0",
+    "structlog-sentry==1.4.0",
+    "sentry-sdk==1.12.0",
+    "analytics-python==1.2.9",
+    # Docs
+    "Pygments==2.13.0",
+    "Markdown==3.4.1",
+    "coreapi==2.3.3",
+    "aiosmtpd==1.4.2",
```

Comment:
the first three here are used for docs but are not in the `docs` group. Also, `aiosmtpd` should be under base deps or perhaps an email group.

Reply:
Yes, but they are not needed for the `docs` step in our CI, right? Thought this would speed things up with `tox-pdm`, but I might be wrong. It's difficult to measure performance though, as the drone cache behaves weirdly. Gonna investigate more.

Regarding `aiosmtpd`, you're right, moving it now.

Reply:
@LudvigHz, are you sure `coreapi` is part of the `docs` group? I don't immediately see the correlation.

Reply:
> DEPRECATION NOTICE: Use of CoreAPI-based schemas were deprecated with the introduction of native OpenAPI-based schema generation as of Django REST Framework v3.10. See the [Version 3.10 Release Announcement](https://www.django-rest-framework.org/community/3.10-announcement/) for more details.

And since you're running `djangorestframework==3.14.0` maybe it would be an idea to just get rid of it 🤷🏻 

https://github.com/webkom/lego/blob/84fc5e728db4fe218d8481a4adf7e87f4a7ecd5b/lego/settings/rest_framework.py#L23

Reply:
Nice catch, @SmithPeder!